src/app/components/Game/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+
+import Board from './components/Board';
+import styles from './styles.scss';
+
+class Game extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
if you just want to set the state here, instead of calling the constructor, you can directly put `state = { ... }`